src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -21,6 +22,8 @@ public class IndexingConfiguration : ConfigurationCategory
         public IndexingConfiguration(RavenConfiguration root)
         {
             _root = root;
+
+            MaximumSegmentMergeSize = new Size(PlatformDetails.Is32Bits ? 128 : 1024, SizeUnit.Megabytes);
```

Comment:
I didn't expect 1GB to be our default (I was thinking about 4GB). Does it mean that even if we limit the merge size significantly it doesn't affect the querying performance much?

Reply:
This is the minimum size that Lucene is considering for segment merging.
I had 10 segments of 3.9GB each which were merged to a single one of ~40GB.
Resulting in tx of 40GB (40GB of scratch files and ~80GB of compression buffer).
This single transaction took about 3 hours to complete.

Reply:
Is `MaximumSegmentMergeSize ` the correct name for that since it's about _min_ (not max)?

Reply:
Sorry, this is the _maximum_ size